dask_saturn/core.py

```diff
@@ -129,7 +130,8 @@ def __init__(
         self._name = self.dask_cluster_id
         self._dashboard_link = info["dashboard_link"]
         self._scheduler_address = info["scheduler_address"]
-        self.loop = None
+        if distributed.__version__ < '2022.08':
```

Comment:
I think this is safer: 

```suggestion
        if parse_version(distributed.__version__) < parse_version('2022.08'):
```